frontend/src/components/UserSearch.vue

```diff
@@ -276,7 +275,11 @@ export default {
   name: "UserSearch",
   data() {
     return {
-      organizations: organizations,
+      organizations: app_config.organizations.map((item) => {
+        item.value = JSON.stringify(item);
+        item.text = `${item.id} - ${item.name}`;
+        return item;
+      }),
```

Comment:
If I don't use `map`, then the organizations drop-down shows the organizations as `Object`. I copied this `map` pattern from `UserDetails`, so now it's consistent too. I don't really understand what necessitates this change when it was working before, but at least it's all consistent now.